README.md

```diff
@@ -32,7 +32,7 @@ news and general information.
 * Run the [Accumulo examples][examples] to learn how to write Accumulo clients
 * View the [Javadocs][javadocs] to learn the Accumulo API
 
-More resources can be found on the [project website][accumulo].
+More resources can be found on the [project website][accumulo].  To commit to the other Accumulo sub projects checkout the [website github][website], [testing], and the [wikisearch] repositories.
```

Comment:
The README tends to get bloated, with unnecessary information. I think we should keep it limited to the scope of giving basic information about the current repo, rather than be used to highlight other projects. I think we can use the website to bring attention to those other projects instead, so we can leave the README focused on this project.